docs/templates/applications.md

```diff
@@ -189,17 +191,17 @@ and a top-5 validation accuracy of 0.945.
 
 Note that this model is only available for the TensorFlow backend,
 due to its reliance on `SeparableConvolution` layers. Additionally it only supports
-the data format "channels_last" (height, width, channels).
+the data format `channels_last` (height, width, channels).
```

Comment:
This is string, not a code keyword (such as a variable name). Please format as a string. Generally avoid any changes that aren't directly related to your PR.

Reply:
Okay, I can change it back.